dev-lang/64tass/64tass-1.57.2900.recipe

```diff
@@ -0,0 +1,66 @@
+SUMMARY="Multi pass optimizing macro assembler for the 65xx series of processors"
+DESCRIPTION="64tass is cross assembler targeting the 65xx series of micro processors \
+which features the familiar syntax of Omicron TASS and TASM."
+HOMEPAGE="https://sourceforge.net/projects/tass64/"
+COPYRIGHT="Daniel Berlin
+	2001 Benjamin Sittler
+	2010 Franck Bui-Huu
+	Alain Magloire
+	2002-2022 Zsolt Kajtar"
+LICENSE="GNU GPL v2
+	GNU LGPL v2
+	GNU LGPL v2.1
+	my_getopt"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/project/tass64/source/64tass-$portVersion-src.zip"
+CHECKSUM_SHA256="e4f0dcb1b00d5206ccb391eb5b8b43f4d95f064d63af739a71f5c822bbfc56a4"
+SOURCE_DIR="64tass-$portVersion-src"
+
+ARCHITECTURES="all ?arm ?ppc ?sparc"
```

Comment:
```suggestion
ARCHITECTURES="all !x86_gcc2"
```
Does the port build with gcc2? If not then it should be excluded from the primary architectures.  The other untested arches can be removed.

Reply:
Yes it does :) When it was packaged last time there were incompatibilities however these were resolved since then. That's one of the reasons why I'd like to have it updated. So I've changed it to "all".

Reply:
I think this is going to cause a conflict on 32 bit systems.  If you install `64tass` and `64tass_x86` then both packages will try to put the commands in /bin

Reply:
Installing both versions is not a good idea as all files would conflict. I've changed it as suggested, the newer GCC makes better code. Who really wants a gcc2 version can just force it.